ko/django_forms/README.md

```diff
@@ -363,13 +358,13 @@ form = PostForm(instance=post)
 축하합니다! 여러분의 어플리케이션이 점점 더 완벽해지고 있어요!
 
 장고 폼에 대해 자세한 정보가 필요하다면 공식 문서를 읽어보세요. :
-https://docs.djangoproject.com/en/1.11/topics/forms/
+https://docs.djangoproject.com/en/2.0/topics/forms/
 
 ## 보안
 
 링크를 클릭해 새로운 포스트가 나오게 만드는 것은 멋진 일이에요! 지금은 웹사이트를 방문하는 누구든지 글을 쓸 수 있지만, 그렇게 하고 싶지 않을 수 있어요. 나에게만 보이고 다른 사람에게는 보이지 않는 버튼을 만들어 볼게요.
 
-`blog/templates/blog/base.html` 파일에서, `page-header` `div`를 찾아 그 위에 아래와 같이 입력합니다 :
+`blog/templates/blog/base.html` 파일에서, `page-header` `div`를 찾아 아래와 같이 작성된 앵커 태그를 찾습니다:
```

Comment:
[Fix Mistranslation]
Not typing, just finding.
Here is original text: `Open blog/templates/blog/base.html in the code editor, find our page-header div and the anchor tag you put in there earlier. It should look like this:`